aiogram/dispatcher/dispatcher.py

```diff
@@ -515,7 +516,6 @@ async def start_polling(
                     )
 
             workflow_data = {
-                "dispatcher": self,
```

Comment:
startup/shutdown hooks should also have access to the Disaptcher, so you can't remove this line.